pages/facts.html

```diff
@@ -4,9 +4,71 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/facts.css">
+  <title>Fun Facts</title>
 </head>
 <body>
-  
+  <header class="header_container">
+    <nav>
+      <ul class="nav_container">
+        <h1>Fun Facts</h1>
+        <li>
+          <a href="../index.html">Home</a>
+        </li>
+        <li>
+          <a href="gallery.html">Memes</a>
+        </li>
+      </ul>
```

Comment:
I'd consider the `<h1>` tag to be a sibling to the `ul` tag because unordered lists should only have list items as children. 

<img width="980" alt="image" src="https://github.com/user-attachments/assets/d3acf4da-97e5-4568-ba64-8b6673250f75" />

```suggestion
      <h1>Fun Facts</h1>
      <ul class="nav_container">
        <li>
          <a href="../index.html">Home</a>
        </li>
        <li>
          <a href="gallery.html">Memes</a>
        </li>
      </ul>
```